src/status_im/communities/core.cljs

```diff
@@ -94,9 +94,17 @@
 
 (defn- handle-my-request
   [db {:keys [community-id state deleted] :as request}]
-  (if (and (= constants/community-request-to-join-state-pending state) (not deleted))
-    (assoc-in db [:communities/my-pending-requests-to-join community-id] request)
-    (update-in db [:communities/my-pending-requests-to-join] dissoc community-id)))
+  (let [{:keys [name]} (get-in db [:communities community-id])]
+    (cond (and (= constants/community-request-to-join-state-pending state) (not deleted))
+          (assoc-in db [:communities/my-pending-requests-to-join community-id] request)
+          (and (= constants/community-request-to-join-state-accepted state) (not deleted))
+          (do (re-frame/dispatch [:toasts/upsert
+                                  {:icon       :correct
+                                   :id         :joined-community
+                                   :icon-color (:positive-01 @colors/theme)
```

Comment:
Not blocking comment: I know this is frustrating to pass the color every time. But, we shouldn't be using the colors from the `status-im` ns as it's old. There is an [issue](https://github.com/status-im/status-mobile/issues/16733) with a refactoring toast to introduce the `type` prop to handle the icon color. This would eventually help us clean up the usage of icon-color.